rust/ql/test/query-tests/security/CWE-770/UncontrolledAllocationSize.expected

```diff
@@ -501,3 +524,5 @@ nodes
 | main.rs:323:27:323:27 | v | semmle.label | v |
 | main.rs:324:25:324:25 | v | semmle.label | v |
 subpaths
+testFailures
+| main.rs:202:32:202:38 | realloc | Unexpected result: Alert=arg1 |
```

Comment:
This result is marked `MISSING` in the source, all we need to do is remove the word `MISSING:` in `main.rs` line 528 and it's a win!

I don't see any other meaningful changes to results of tests.  :tada: